examples/cute/tutorial/sgemm_1_sycl.cpp

```diff
@@ -39,6 +39,26 @@
 #include "cutlass/util/sycl_event_manager.hpp"
 #include "cutlass/util/GPU_Clock.hpp"
 
+namespace syclcompat {
+  template <class F, int Dim>
+  sycl::event launch(const sycl::nd_range<Dim> &range, sycl::queue q, const F& f) {
+    return q.parallel_for(detail::transform_nd_range<Dim>(range),  [=](sycl::nd_item<Dim>) { f(); });
+  }
+  template <class F, int Dim>
+  sycl::event launch(const sycl::nd_range<Dim> &range, const F& f) {
+    return launch(range, get_default_queue(), f);
+  }
+  // Alternative launch through dim3 objects
+  template <class F>
+  sycl::event launch(const dim3 &grid, const dim3 &threads, sycl::queue q, const F& f) {
+    return launch(sycl::nd_range<3>{grid * threads, threads}, q, f);
+  }
+  template <class F>
+  sycl::event launch(const dim3 &grid, const dim3 &threads, const F& f) {
+    return launch(grid, threads, get_default_queue(), f);
+  }
+}
```

Comment:
Shall we put those as PR on sycl::compat repo

Reply:
https://github.com/intel/llvm/pull/18021